src/Backend/Core/Installer/Data/install.sql

```diff
@@ -11,7 +11,7 @@ CREATE TABLE IF NOT EXISTS `meta` (
   `custom` text CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci COMMENT 'used for custom meta-information',
   `data` text COMMENT 'used for extra meta-information',
   PRIMARY KEY (`id`),
-  KEY `idx_url` (`url`)
+  KEY `idx_url` (`url`(249))
```

Comment:
I've read on the big internet that 249 was more safe, but not sure if this is right, since I can't reproduce this issue (probably because of other limits in mariadb).